modules/@apostrophecms/i18n/i18n/sk.json

```diff
@@ -0,0 +1,381 @@
+  "documentPublishedNoPreview": "Dokument bol zverejnený, ale nepodarilo sa prejsť na ukážku.",
+  "documentSaved": "Dokument uložený.",
+  "doesNotExistYet": "Zatiaľ neexistuje",
+  "doesNotExistYetDescription": "Dokument zatiaľ neexistuje ako {{ mode }} v jazykovej mutácii {{ locale }}.",
+  "download": "Stiahnuť",
+  "draft": "Návrh",
+  "draftDeleted": "Návrh odstánený",
+  "draftDiscarded": "Návrh vyradený",
+  "draftSaved": "Návrh uložený",
+  "draftSavedNoPreview": "Návrh bol uložený, ale nepodarilo sa prejsť na ukážku.",
+  "dropMedia": "Sem vložte nové médiá",
+  "duplicate": "Duplikovať...",
+  "edit": "Upraviť",
+  "editType": "Upraviť {{ type }}",
+  "editor": "Editor",
+  "editRelationshipFor": "Upraviť vzťah pre {{ title }}",
+  "editWidget": "Upraviť prvok stránky",
+  "editWidgetForeignTooltip": "Kliknutím upravíte tento obsah v jeho prirodzenom kontexte",
+  "editWidgetType": "Upraviť {{ label }}",
+  "email": "Email",
+  "emptyRichTextWidget": "Prvok s prázdnym textom",
+  "enabled": "Povolené",
+  "error": "Nastala chyba",
+  "errorCount": "{{ count }} chyba zostáva",
+  "errorCount_plural": "{{ count }} zostávajúcich chýb",
+  "errorCreatingNewContent": "Chyba pri vytváraní nového, prázdneho obsahu.",
+  "errorWhileRestoring": "Pri obnove predtým publikovanej verzie sa vyskytla chyba.",
+  "errorWhileUnpublishing": "Pri rušení zverejnenia dokumentu sa vyskytla chyba.",
+  "errorWhileArchiving": "Pri presúvaní dokumentu do archívu sa vyskytla chyba.",
+  "errorWhileDismissing": "Pri odsielaní sa vyskytla chyba",
+  "errorWhileDismissingDescription": "Pri zrušení odsielania príspevku sa vyskytla chyba.",
+  "errorWhilePublishing": "Pri publikovaní sa vyskytla chyba",
+  "errorWhilePublishingDocument": "Pri publikovaní dokumentu sa vyskytla chyba.",
+  "errorWhilePublishingParentPage": "Pri publikovaní nadradenej stránky sa vyskytla chyba.",
+  "errorWhileRestoringArchive": "Pri obnove dokumentu z archívu sa vyskytla chyba.",
+  "errorWhileRestoringPrevious": "Pri obnove predtým publikovanej verzie sa vyskytla chyba.",
+  "errorWhileSubmitting": "Pri odosielaní sa vyskytla chyba",
+  "errorWhileSubmittingDescription": "Pri odosielaní dokumentu sa vyskytla chyba.",
+  "errorPageMessage": "Došlo k chybe",
+  "errorPageStatusCode": "500",
+  "errorPageTitle": "Došlo k chybe v nadpise stánky",
+  "everythingElse": "Všetko ostatné",
+  "exit": "Odísť",
+  "fetchPublishedVersionFailed": "Pri načítaní zverejnenej verzie dokumentu sa vyskytla chyba.",
+  "fieldHasUnpublishedChanges": "Toto pole obsahuje nepublikované zmeny",
+  "file": "Súbor",
+  "fileTag": "Značka súboru",
+  "fileTags": "Značky súboru",
+  "fileTypeNotAccepted": "Typ súboru nebol prijatý. Povolené rozšírenia: {{ extensions }}",
+  "fileInvalid": "Súbor nebol prijatý. Môže byť poškodený alebo sa jeho obsah nemusí zhodovať s príponou súboru.",
+  "files": "Súbory",
+  "filter": "Filter",
```

Comment:
@Miselrkba Is this the correct translation for "filter" as a verb? Since it's the same as English I looked it up and saw this might be the noun in Slovak. I see "filtrovať" as a verb, but that might not be better than this for the UI.


![Screen Shot 2021-09-23 at 4 26 14 PM](https://user-images.githubusercontent.com/1155984/134586385-732787a7-ebf1-4f41-935d-eea2828cd3da.png)

Reply:
Filter is a noun. So "filtrovať" would be more appropriate in this case. Well spotted 👍